Can't Compile without Odin Inspector

Getting:
```
Library\PackageCache\com.graphene.core@549b35e32b\Core\Scripts\Rendering\Renderer.cs(105,6): error CS0246: The type or namespace name 'Sirenix' could not be found (are you missing a using directive or an assembly reference?)
```

--- a/src/Core/Scripts/Rendering/Renderer.cs
+++ b/src/Core/Scripts/Rendering/Renderer.cs
@@ -101,8 +101,12 @@ namespace Graphene
       if (model is IModel iModel)
         iModel.onModelChange?.Invoke();
     }
-
+    
+#if ODIN_INSPECTOR
     [Sirenix.OdinInspector.Button]
+#elif NAUGHTY_ATTRIBUTES
+    [NaughtyAttributes.Button]
+#endif
     public void HardRefresh()
     {
       RenderToContainer(GetDrawContainer());

